.github/workflows/release.yaml

```diff
@@ -5,7 +5,9 @@ on:
       - master
 
 permissions:
-  contents: read
+  contents: write # to be able to publish a GitHub release
+  issues: write # to be able to comment on released issues
```

Comment:
Do we use this at this moment, or do we have plans to?

Reply:
yes we do `@semantic-release/github` comments on PRs to list in what version they are included

Reply:
the separate `issues` permission is used when an issue is linked to a PR that made it into the release. It would then also comment on the linked issue

Reply:
@sparten11740 ACK, thank you for the elaboration